### PR TITLE
Location feature

### PR DESCRIPTION
**Pull request by skairali**

### CODE REVIEW

**Comment by skairali** [comment]
@delliott90   Please review `python3 main.py transmit security_advisor '{"host" :"https://us-south.secadvisor.cloud.ibm.com/findings/v1"}' '{"auth": { "accountID" :"<<>>" ,"apiKey" : "<<>" }}' results "[url:value = 'https://github.ibm.com//issues/9340'] START t'2021-02-20T20:27:40.084201Z' STOP t'2021-02-22T20:27:40.084201Z'" 0 3.  `     This is how the current transmission happens where user need to provide host.  As per the new change.  `python3 main.py transmit security_advisor '{"auth": { "accountID" :"<<>>" ,"apiKey" : "<<>" }}' results "[url:value = 'https://github.ibm.com//issues/9340'] START t'2021-02-20T20:27:40.084201Z' STOP t'2021-02-22T20:27:40.084201Z'" 0 3` should also work without **host** parameter

File: stix_shifter_modules/security_advisor/stix_transmission/auth.py [hunk 1]
```diff
@@ -25,4 +26,15 @@ def obtainAccessToken(self):
             return response.json()["access_token"]
 
         else:
-            raise Exception("Authorizaion Failed")
\ No newline at end of file
+            raise Exception("Authorizaion Failed")
+
+    def find_location(self, accountID, host):
+        if host == "":
+            adminApiUrl = os.getenv('ADMIN_API_URL', 'https://compliance.cloud.ibm.com/admin/v1')
```
Comment (delliott90) [inline stix_shifter_modules/security_advisor/stix_transmission/auth.py:33]: You may want to define `https://compliance.cloud.ibm.com/admin/v1` in a constant at the top just so it's easier to find if that default API url changes in the future.

File: stix_shifter_modules/security_advisor/stix_transmission/auth.py [hunk 2]
```diff
@@ -25,4 +26,15 @@ def obtainAccessToken(self):
...
+            headers = {"Authorization": "Bearer {0}".format(self.obtainAccessToken())}
+            resp = requests.get("{0}/accounts/{1}/settings".format(adminApiUrl, accountID), headers=headers)
+            current_location_id = resp.json()["location"]["id"]
```
Comment (delliott90) [inline stix_shifter_modules/security_advisor/stix_transmission/auth.py:36]: Is there any danger that either location or id won't be in the `resp` object? If so you'll want to handle this potential error.

File: stix_shifter_modules/security_advisor/stix_transmission/auth.py (hunk 1 above)
Comment (skairali) [reply stix_shifter_modules/security_advisor/stix_transmission/auth.py]: Done

File: stix_shifter_modules/security_advisor/stix_transmission/auth.py (hunk 2 above)
Comment (skairali) [reply stix_shifter_modules/security_advisor/stix_transmission/auth.py]: Done

**Review by delliott90 — APPROVED** [review]

**Pull request merged**